src/structs/quantization_tables.rs

```diff
@@ -44,6 +50,9 @@ impl QuantizationTables {
             self.quantization_table[i] = quantization_table[RASTER_TO_JPEG_ZIGZAG[i] as usize];
         }
 
+        self.quantization_table_transposed =
+            AlignedBlock::new(cast(i16x8::transpose(cast(self.quantization_table))));
```

Comment:
For my knowledge - what is transpose of single dimensional array? It's not just rev, right?